### ISSUE TITLE
Bowel sounds high pitched

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10116]
**Preferred term label:**
Bowel sounds high pitched

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0030142

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Ogilvie Syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000633
- label: High-pitched bowel sounds
- parents: [Abnormal bowel sounds (HP:0030142)](https://hpo.jax.org/app/browse/term/HP:0030142)
- pmid: PMID:30252358
- def: Abnormal bowel sounds of an unually high frequency.
- comment: This finding is characteristic of Ogilvie syndrome.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10116
- synonyms:
